### PR TITLE
feat: Add support for milestones, socials, tags

### PR DESCRIPTION
**Pull request by Pradumnasaraf**
####  Changes proposed

- Added support for `milestones`, `socials`, and `tags`
- Change naming for JSON structure from `sampleJson ` to `jsonSchema` in `src/createjson/helper/createUser.js`
- Format all files (run prettier)

####  Note to reviewers

Hey, @Ananya2001-an,

Feel free to review it when you feel comfortable and free.

### CODE REVIEW

**Comment by Ananya2001-an** [comment]
Sure I will review it soon 👍

File: src/shared/questions/milestones.js [hunk 1]
```diff
@@ -0,0 +1,74 @@
+const { prompt } = require("enquirer");
+const { geticons } = require("../assets/icons");
+
+let milestones = [];
+
+async function addmilestones(bool) {
+  icons = await geticons();
```
Comment (Ananya2001-an) [inline src/shared/questions/milestones.js:7]: need to declare icons array here

File: src/shared/questions/socials.js [hunk 2]
```diff
@@ -0,0 +1,52 @@
+const { prompt } = require("enquirer");
+const { geticons } = require("../assets/icons");
+
+let socials = [];
+
+async function addsocials(bool) {
+  icons = await geticons();
```
Comment (Ananya2001-an) [inline src/shared/questions/socials.js:7]: same here as above

File: src/shared/questions/socials.js [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+
+  while (bool) {
+    icons = await geticons();
```
Comment (Ananya2001-an) [inline src/shared/questions/socials.js:10]: remove this line we already have called the getIcons fn

File: src/shared/questions/milestones.js (hunk 1 above)
Comment (Pradumnasaraf) [reply src/shared/questions/milestones.js:8]: Will do it. Thank you

File: src/shared/questions/socials.js (hunk 3 above)
Comment (Pradumnasaraf) [reply src/shared/questions/socials.js]: Will do it. Thank you

File: src/shared/questions/socials.js (hunk 2 above)
Comment (Pradumnasaraf) [reply src/shared/questions/socials.js:8]: Will do it. Thank you

**Comment by Pradumnasaraf** [comment]
Hey, 

Thanks for the review. I have done the requested changes.

**Review by Ananya2001-an — APPROVED** [review]
Awesome

**Comment by Ananya2001-an** [comment]
You can merge it now

**Comment by Pradumnasaraf** [comment]
> You can merge it now

Thank you. DId it

**Pull request merged**